Replace zapi references with zed (#6)
brimdata/cloud: README.md

You can access your Zed lake in the Brim cloud using
* from your desktop using [the Brim app](#the-brim-app)

> * from the command-line using [zed CLI command](#the-cli)

Reviewer: ```suggestion
* from the command-line using [`zed` CLI command](#the-cli)
```

### The CLI

The `zed` command lets you access all of the functionality exposed to the Brim
App on the command line. Type `zed -h` to get online help.

> The quickest way to get the `zed` cli on macOS, Linux, or Windows is to download
> a pre-built release binary. You can find these binaries on the [GitHub releases](https://github.com/brimdata/zed/releases)
> page.

Reviewer: ```suggestion
The quickest way to get the `zed` command on macOS, Linux, or Windows is to [download
a pre-built release](https://github.com/brimdata/zed/releases).
```
